Agents/utils/python-utils/twautils/test/twa_logging_demo.py

```diff
@@ -0,0 +1,27 @@
+from twautils.log import twa_logging
+
+
+def demo():
+	"""
+		Demo the logging functionality.
+	"""
+	print("=== Development Logging ===")
+	dev_logger = twa_logging.get_logger("dev")
+	dev_logger.debug("This is a DEBUG statement")
+	dev_logger.info("This is an INFO statement")
+	dev_logger.warning("This is a WARNING statement.")
+	dev_logger.error("This is an ERROR statement.")
+	dev_logger.critical("This is a CRITICAL statement.")
+
+	print("=== Production Logging ===")
+	prod_logger = twa_logging.get_logger("prod")
```

Comment:
... and this would be:
```suggestion
	prod_logger = twautils.log.get_logger("prod")
```

Reply:
Updated.